documentation/book/con-resource-RBAC.adoc

```diff
@@ -0,0 +1,12 @@
+[id='provision-rbac-{context}']
+= Provision Role-Based Access Control for the Cluster Operator
+
+For the Cluster Operator to function it needs permission within the {ProductPlatformName} cluster to interact with the resources it manages and interacts with. These include the desired resources, such as  `Kafka`, `Kafka Connect`, and so on, as well as the managed resources, such as `ConfigMaps`, `Pods`, `Deployments`, `StatefulSets`, `Services`, and so on. Such permission is described in terms of {ProductPlatformName} role-based access control (RBAC) resources:
```

Comment:
Procedure missing. How does customer configure CO to have access to resources managed by {ProductPlatformName}?